pyproject.toml

```diff
@@ -71,7 +71,7 @@ packages = [{ include = "litestar" }]
 python = ">=3.8,<4.0"
 alembic = { version = "*", optional = true }
 annotated-types = { version = "*", optional = true }
-anyio = ">=3"
+anyio = ">=3,<4"
```

Comment:
I had to pin this because if not CI fails: tests/helpers.py:9: in <module>
    from anyio._core._compat import _ContextManagerWrapper
E   ModuleNotFoundError: No module named 'anyio._core._compat' in here for instance (https://github.com/litestar-org/litestar/actions/runs/6070435740/job/16466493190?pr=2250)

Reply:
tests are ok except some other CI stuff unrelated

Reply:
ok I'll merge main once that settled :)